python/cucim/src/cucim/skimage/segmentation/_chan_vese.py

```diff
@@ -426,15 +426,15 @@ def chan_vese(
         image = image / cp.max(image)
 
     i = 0
+    mu = cp.asarray(mu, dtype=float_dtype)
+    lambda1 = cp.asarray(lambda1, dtype=float_dtype)
+    lambda2 = cp.asarray(lambda2, dtype=float_dtype)
```

Comment:
Should we move this and all other assignment right after the normalization of the `image` input above?

Reply:
sure can move it all there